src/components/Signup.vue

```diff
@@ -141,6 +169,12 @@ export default {
       FileSaver.saveAs(blob, filename)
 
       this.downloaded = true
+    },
+
+    addAddressToWhiteList (address) {
```

Comment:
What about validation? As far as I remember, we had a method to validate ETH addresses here https://github.com/d3ledger/back-office/blob/develop/src/components/mixins/inputValidation.js and example of usage here https://github.com/d3ledger/back-office/blob/develop/src/components/Wallets/Wallet.vue

I guess we can skip it for now and create a task for validation later on.